src/components/ToDoItem/ToDoItem.js

```diff
@@ -41,30 +50,48 @@ export const ToDoItem = (message) => {
       return;
```

Comment:
Why this method which is called checkboxClick(should be handleOnCheckboxClick) have eventListener on "blur" ?? , we only want to react here on "change" input type:checkbox. This is to be refactored

Reply:
.